### ISSUE TITLE
GPS Integration with GLIM

### ISSUE THREAD

**Issue by Shaswat2001** [koide3/glim#123]
Hi, 

I was just wondering if there has been any developments on integrating GPS data with SLAM algorithm?

Would really appreciate if someone can guide me as to how I can integrate GPS data as well. 

Regards
Shaswat

**Comment by eandert** [comment]
I agree, it would be incredible if GLIM integrated GPS as an option for input.
